### PR TITLE
Hair 66 store reviews backend

### PR DESCRIPTION
**Pull request by cynthiaeddy**
pr for capturing review in frontend and storing in db to the reviews, businesses, and users collections.

review in frontend for Stone Cutz:
Screen Shot 2021-12-15 at 2 44 05 PM
reviews collection (bottom review): _id:ObjectId(" 61ba458a1931834a109962ab")
Screen Shot 2021-12-15 at 2 45 13 PM
businesses collection:  reviews: Array 1: ObjectId(" 61ba458a1931834a109962ab")
Screen Shot 2021-12-15 at 2 46 09 PM
users collection:  reviews: Array 7: ObjectId(" 61ba458a1931834a109962ab")
Screen Shot 2021-12-15 at 2 46 47 PM

### CODE REVIEW

File: controllers/businesses-controller.js [hunk 1]
```diff
@@ -77,7 +76,6 @@ const getBusinesses = async (req, res, next) => {
   })
 }
 
-// Update Business
```
Comment (CodeMeKathy) [inline controllers/businesses-controller.js:80]: Was this deletion intentional?

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "main": "app.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon app.js",
+    "start": "node app.js",
+    "dev": "nodemon server.js",
```
Comment (CodeMeKathy) [inline package.json]: Hi, @cynthiaeddy! There is no server.js file.

**Review by CodeMeKathy — CHANGES_REQUESTED** [review]
Hi, @cynthiaeddy.  Great work; thank you for your efforts!  However, I am requesting changes and suggesting one minor change, as we discussed.  I suggest modifying or removing the changes made to the `package.json` file as there is no `server.js` file.

File: controllers/businesses-controller.js (hunk 1 above)
Comment (cynthiaeddy) [reply controllers/businesses-controller.js:80]: probably not - put back in

File: package.json (hunk 2 above)
Comment (cynthiaeddy) [reply package.json]: resolved

**Comment by cynthiaeddy** [comment]
@CodeMeKathy hi! I updated package.json, so I guess you need to approve before I can merge?

**Review by CodeMeKathy — APPROVED** [review]
Thanks for making the changes @cynthiaeddy!! Happy New Year!!

**Comment by cynthiaeddy** [comment]
@CodeMeKathy  pr merged. happy new year to you too!!

**Pull request merged**